### DISCUSSION TITLE
have multiSelection without shift key press

### DISCUSSION THREAD

**Discussion by fatemeh-amerehi** [fabricjs/fabric.js#7460] [Q&A]
hi... i'm trying to have multiSelection without shift key couse my project is web Application and shift key in phone ...
i tried a lot of way's but all had some issues
any solution for this?

**Comment by ShaMan123** [comment]
imperative `canvas.setActiveObject`

**Comment by fatemeh-amerehi** [comment]
> imperative `canvas.setActiveObject`

i tried this but got coords and getRetinaScaling errors

**Comment by ShaMan123** [comment]
Sounds like you are doing something wrong.
Does the object have a reference to canvas?
